android/core/src/main/java/com/stadiamaps/ferrostar/core/annotation/valhalla/ValhallaOSRMExtendedAnnotation.kt

```diff
@@ -0,0 +1,11 @@
+package com.stadiamaps.ferrostar.core.annotation.valhalla
+
+import com.squareup.moshi.Json
+import com.stadiamaps.ferrostar.core.annotation.Speed
+
+data class ValhallaOSRMExtendedAnnotation(
+    @Json(name = "maxspeed") val speedLimit: Speed?,
+    val speed: Double?,
```

Comment:
Is this double still needed? I think this was initial pass note, but probably not relevant now that `Speed` exists.

Reply:
I was checking, on iOS there is:

```swift
    /// The speed limit of the segment.
    public let speedLimit: MaxSpeed?

    /// The estimated speed of travel for the segment, in meters per second.
    public let speed: Double?
```

should I comment them as such and leave it instead?

Reply:
I added the comments.